src/plugins/intel_cpu/src/nodes/mvn.cpp

```diff
@@ -2047,15 +2047,18 @@ void MVN::prepareParams() {
 #endif
 
     auto selectedPD = getSelectedPrimitiveDescriptor();
-    mvnAttrs.src_prc = selectedPD->getConfig().inConfs[0].getMemDesc()->getPrecision();
-    mvnAttrs.dst_prc = selectedPD->getConfig().outConfs[0].getMemDesc()->getPrecision();
-    if (getParentEdgeAt(0)->getMemory().getDesc().hasLayoutType(LayoutType::ncsp)) {
+    auto inMemDesc = selectedPD->getConfig().inConfs[0].getMemDesc();
+    auto outMemDesc = selectedPD->getConfig().outConfs[0].getMemDesc();
+    mvnAttrs.src_prc = inMemDesc->getPrecision();
```

Comment:
As part of this PR I also propose to move other calculations to the load stage such as `mvnAttrs` preparing (`src_prc`, `dst_prc`, `layout`), we know all the information at the load stage so let's move this calculation to  `createPrimitive()` method as well.

Reply:
good catch, done, thanks!